README.md

```diff
@@ -24,6 +24,20 @@ gulp.task('sass', function () {
 });
 ```
 
+You can also compile synchronously, doing something like this:
+
+```javascript
+var gulp = require('gulp');
+var gutil = require('gulp-util');
```

Comment:
What do you mean I'm not? Line 36 `.pipe(sass.sync().on('error', sass.logError))`